estuary_updater/handlers/base.py

```diff
@@ -88,6 +88,24 @@ def is_container_build(self, build_info):
         else:
             return False
 
+    def is_module_build(self, build_info):
+        """
+        Check whether a Koji build is a module build.
+
+        :param KojiBuild build_info: build info from Teiid
+        :return: boolean value indicating whether the build is a module build
+        :rtype: bool
+        """
+        try:
+            extra_json = json.loads(build_info['extra'])
```

Comment:
The Koji API should be returning this as a dictionary already.

Reply:
Wouldn't we want to leave it this way though to make the method more versatile, or will this never occur?  I had it this way because of how it was written in estuary api